patches/amd-mainline/hipBLASLt/0001-Skip-clock-reset-if-rocm-smi-is-not-available.patch

```diff
@@ -0,0 +1,65 @@
+From 6d884cba70257255dc6776372292b93cf1a1a4cb Mon Sep 17 00:00:00 2001
+From: Stella Laurenzo <stellaraccident@gmail.com>
+Date: Tue, 29 Apr 2025 17:48:40 -0700
+Subject: [PATCH 1/2] Skip clock reset if rocm-smi is not available.
+
+* Pinning clocks was already conditioned on this, so the cleanup logic should also be.
+* Also removes a windows.h `#define` that was causing trouble.
+---
+ .../rocblaslt/src/rocblaslt_auxiliary.cpp        |  3 +++
+ tensilelite/Tensile/Common/GlobalParameters.py   | 16 +++++++++++++---
+ 2 files changed, 16 insertions(+), 3 deletions(-)
+
+diff --git a/library/src/amd_detail/rocblaslt/src/rocblaslt_auxiliary.cpp b/library/src/amd_detail/rocblaslt/src/rocblaslt_auxiliary.cpp
+index db0d3d30..bee22eb7 100644
+--- a/library/src/amd_detail/rocblaslt/src/rocblaslt_auxiliary.cpp
++++ b/library/src/amd_detail/rocblaslt/src/rocblaslt_auxiliary.cpp
+@@ -28,6 +28,9 @@
+ #include <Windows.h>
+ #include <io.h>
+ #include <libloaderapi.h>
++
++// Remove defines that conflict locally.
++#undef CONST
+ #else
+ #define _GNU_SOURCE
+ #include <dlfcn.h>
+diff --git a/tensilelite/Tensile/Common/GlobalParameters.py b/tensilelite/Tensile/Common/GlobalParameters.py
+index 3efc6209..a01633c9 100644
+--- a/tensilelite/Tensile/Common/GlobalParameters.py
++++ b/tensilelite/Tensile/Common/GlobalParameters.py
+@@ -549,7 +549,14 @@ def assignGlobalParameters(config, isaInfoMap: Dict[IsaVersion, IsaInfo]):
+         globalParameters["CmakeCCompiler"] = os.environ.get("CMAKE_C_COMPILER")
+ 
+     globalParameters["ROCmBinPath"] = os.path.join(globalParameters["ROCmPath"], "bin")
+-    globalParameters["ROCmSMIPath"] = locateExe(globalParameters["ROCmBinPath"], "rocm-smi")
++    try:
++        globalParameters["ROCmSMIPath"] = locateExe(globalParameters["ROCmBinPath"], "rocm-smi")
++    except OSError:
++        if os.name == "nt":
++            # rocm-smi is not presently supported on Windows so do not require it.
++            pass
++        else:
++            raise
```

Comment:
Before we had
https://github.com/ROCm/TheRock/blob/817950a0d2391d43922397fcd0cb5972c31d80ff/patches/amd-mainline/hipBLASLt/0007-Skip-clock-reset-if-rocm-smi-is-not-available.patch#L22-L23

Reply:
Yes, I adapted thst to get this. Just lost it in the shuffle. Kkyang is discussing what the right thing to do here is on the upstream patch